core/app/modules/kit-library/assets/js/components/kit-list-item.js

```diff
@@ -0,0 +1,45 @@
+import { Card, CardHeader, CardBody, Heading, CardImage, CardOverlay, Grid, Button } from '@elementor/app-ui';
+import Kit from '../models/kit';
+
+import './kit-list-item.scss';
+
+export default function KitListItem( props ) {
+	return (
+		<Card>
+			<CardHeader>
+				<Heading
+					tag="h3"
+					title={ props.model.title }
+					variant="h5"
+					className="eps-card__headline"
+				>
+					{ props.model.title }
+				</Heading>
+			</CardHeader>
+			<CardBody>
+				<CardImage alt={ props.model.title } src={ props.model.thumbnailUrl || '' }>
+					<CardOverlay>
+						<Grid container direction="column" className="e-kit-library__kit-item-overlay">
+							<Button
+								className="e-kit-library__kit-item-overlay-overview-button"
+								text={ __( 'Overview', 'elementor' ) }
+								icon="eicon-zoom-in-bold"
+								url={ `/kit-library/${ props.model.id }` }
+							/>
+							<Button
+								className="e-kit-library__kit-item-overlay-demo-button"
+								text={ __( 'View Demo', 'elementor' ) }
+								icon="eicon-preview-medium"
+								url={ `/kit-library/${ props.model.id }/preview` }
```

Comment:
Maybe like the site editor format? `#/kit-library/preview/${ id }`

Reply:
The current approach seems more expressive for me, but if consistency is important here, let me know and I will change it.

Reply:
It's consistency and also easier to manipulate.